### ISSUE TITLE
PageSettings > Login Required Reflects across all locales

### ISSUE THREAD

**Issue by Ramakrishnan-Sathyanarayanan** [apostrophecms/apostrophe#3452]
Apostrophe 2.102.3

## To Reproduce

Step by step instructions to reproduce the behavior:

> 1. Go to Home page
> 2. Click on Page Settings > Permissions > Who Can View this page? > LoginRequired
> 3. Save the Settings
> 4. This Setting change is reflected across all the locales of the Home Page

## Expected behavior
> A change in the Login Required in pages between Locales should be independent

## Describe the bug
> Login Required changed in one locale page is reflected across all the locale pages. Ideally we were expecting it to reflect only in one page.

## Details

**Version of Node.js:**
10.24.1

**Server Operating System:**
Amazon Linux 2

**Additional context:**
> Add any other context about the problem here. If the problem is specific to a browser, OS or mobile device, specify which.

**Screenshots**
![image](https://user-images.githubusercontent.com/65652388/134968962-a74de336-5baa-4be0-9126-371b6dfc1412.png)

**Comment by Ramakrishnan-Sathyanarayanan** [comment]
Permissions tab in Page Settings is updated based on workflowGuid(all locale pages) contradicting to other tabs in the same Page Settings(page specific). There are no note/warning mentioning this and it is kind of misleading.

https://github.com/apostrophecms/apostrophe-workflow/blob/0d14379f00ad05dc894e4275af11be99fa3aad2a/lib/callAll.js#L303

**Comment by boutell** [comment]
I agree that from a UX standpoint it would be better if we made this clear, but all permissions for pages do indeed cascade across all locales, as this allows a significant simplification of the permissions interface: what you can do is a combination of two separate things, what documents you're allowed to edit and what locales you're allowed to edit. That won't be changing, but I agree it would be helpful to document it here for the user editing the page settings, so I'll keep this ticket open with a new title. PRs welcome on this.
